python/metatensor-torch/metatensor/torch/atomistic/documentation.py

```diff
@@ -111,8 +111,13 @@ def known_data(self) -> List[str]:
 class NeighborsListOptions:
     """Options for the calculation of a neighbors list"""
 
-    def __init__(self, model_cutoff: float, full_list: bool):
-        ...
+    def __init__(self, model_cutoff: float, full_list: bool, requestor: str = ""):
```

Comment:
Below you write that there is a list of requestors and a new one can be added with `add_requestor`. Why do we need this initial argument `requestor` wouldn't it be a bit more consistent to have an API like

```python
ns = NeighborsListOptions(model_cutoff=5, full_list=False)
ns. add_requestor("my_module")
```

Reply:
Okay, for a compromise we could add a comment stating like:

```rst
:param requestor: Who requested this neighbors list. You can add additional requestors
     later using :meth:`add_requestor`.
```